### ISSUE TITLE
NFTicket Brainstorm

### ISSUE THREAD

**Issue by DanM3rcurius** [SoU-DisCo-of-Urgency/Documentation#4]
![Screenshot 2021-10-14 at 23 48 50](https://user-images.githubusercontent.com/80257662/137400619-e41898c2-bb4a-4636-91d2-d3cdf1b43f52.png)

## To Do's
create PolicyID @Dan/Quinn maybe
serialise art, close after 2 epochs, allow DoU wallet to mint
write metadata @Dan/Quinn maybe
optional (depending what incentive is chosen):
entry check: serial no of NFT (manual o automatic?) @Anna/Dan
setup (UWS) exclusive stream access with Ticket @Anna/Dan
setup discount plugin (shopify & wordpress plugin)

**Comment by DanM3rcurius** [comment]
@evolayak like so?

**Comment by DanM3rcurius** [comment]
### Name of NFT: 
FTR21NFTicket + ID | Example: FTR21NFTicket#022
### Description:
- Incentives overview
### Website:
- link to conference website

### Incentives
This NFTicket is forever yours, you can use the video animation in whatever non financial context you like. It also gives You access to the following 
- the Envisioning Free Spaces (Frei|t|räume) 2021 conference and afterparty
- discounts on future conferences
- NFT-Art drops in 2022
- exclusive content from artist Pedro Marun

Additionally any NFTicket buyer will experience the feeling of having founded the first social activism stake pool on Cardano!

**Comment by DanM3rcurius** [comment]
![Screenshot 2021-11-20 at 04 46 42](https://user-images.githubusercontent.com/80257662/142713429-76b89391-f013-47b1-a24c-464b1355d2e4.png)
NFTicket landing page mockup

**Comment by DanM3rcurius** [comment]
@evolayak @LucoSOU Thoughts?

**Comment by evolayak** [comment]
I would change a couple things but I need to be behind my laptop to do so

**Comment by DanM3rcurius** [comment]
> I would change a couple things but I need to be behind my laptop to do so Sent from ProtonMail for iOS

ok kewl!

**Comment by evolayak** [comment]
How do I edit this? Is it a separate Miro board?

Thanks :)

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by DanM3rcurius** [comment]
It's theFTR21 Miro Board, you can also edit the file in Git https://github.com/SoU-DisCo-of-Urgency/Documentation/blob/main/Website-Content/NFTicket.md
if you fork the repo before (in case you haven't already).

I'll be online in about 2 hours again for a bit then back from 9 or 10pm onwards for the rest of the night

Warmest Regards,
Dan

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by evolayak** [comment]
Okee. I’ll be offline most of today; need a detox 🌈

**Comment by LucoSOU** [comment]
> 

Hi there, still possible to suggest edits to this text? FYI I dont have FTR miro link anymore, if you can send would be fab!

**Comment by evolayak** [comment]
I still need to do my edits

**Comment by evolayak** [comment]
Okay forked my minor edits

**Comment by LucoSOU** [comment]
@DanM3rcurius left a comment with suggest text on the miro. :)

**Comment by DanM3rcurius** [comment]
> @DanM3rcurius left a comment with suggest text on the miro. :)

Yep, that's the same text in here. GitHub should be the single source of truth for documents and copywriting content. That way you never loose it and can track contributions precisely.

**Comment by DanM3rcurius** [comment]
@evolayak here's the miro board link https://miro.com/app/board/o9J_lrN7duQ=/
